core/src/test/java/de/evoila/cf/broker/service/impl/AsyncDeploymentServiceImplTest/BaseTest.java

```diff
@@ -42,16 +44,30 @@
     AsyncDeploymentServiceImpl asyncDeploymentService;
 
     @BeforeEach
-    void setUp()
-    {
+    void setUp() {
         asyncDeploymentService = new AsyncDeploymentServiceImpl(jobProgressService);
     }
 
-    void mockSuccessfulStartJob(String jobProgress)
-    {
+    void mockSuccessfulStartJob(String jobProgress) throws ServiceBrokerException {
         when(serviceInstance.getId())
                 .thenReturn(SERVICE_INSTANCE_ID);
         when(jobProgressService.startJob(eq(JOB_PROGRESS_ID), eq(SERVICE_INSTANCE_ID), anyString(), eq(jobProgress)))
                 .thenReturn(startedJob);
     }
+
+    void mockStartJobThrowsServiceBrokerException(String jobProgress, String description) throws ServiceBrokerException {
```

Comment:
Is there a reason for two separate mockStartJobThrows... methods? The exception could be a parameter.

Reply:
fixed